overcast_to_sqlite/html/page.py

```
@@ -26,6 +27,57 @@ def _convert_urls_to_links(text: str) -> str:
     return "".join(result)
 
 
+class HTMLTagFixer(HTMLParser):
+    SELF_CLOSING_TAGS = {'br', 'hr', 'img', 'input', 'meta', 'link', 'area', 'base', 'col', 'embed', 'source', 'track', 'wbr'}
```

Ruff (0.11.9)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)